src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollPoll.test.ts

```diff
@@ -438,4 +441,46 @@ describe('TransformOktaVerifyEnrollPoll Tests', () => {
     expect((layoutFive.elements[3] as DescriptionElement).options.content)
       .toBe('oie.enroll.okta_verify.setup.skipAuth.canBeClosed');
   });
+
+  it('Setup OV button should use APP_LINK challenge method when selectedChannel is samedevice and platform is android',
```

Comment:
Haven't looked at our same device OV TestCafe specs, do we have any coverage for clicking the setup button and asserting on the behavior? If not, does it seem feasible?

Reply:
No looks like we dont have coverage for that, since I think theres no way we can mock any useful response to test it.

Reply:
But I do see some assertions to test the underlying `setupOVUrl`, that are not being included for gen 3 like for example [here](https://github.com/okta/okta-signin-widget/blob/c53ed6cfcc3248277e4d7d7a3265d47a683d171e/test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js#L1174).  I should be able to get these unblocked for gen 3.

Reply:
@trevoring-okta I updated the coverage in the testcafe test to check the behavior on clicking the setup OV button, similar to how we do it in test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

Reply:
@leemchale-okta LGTM, thanks for adding coverage here since I realize it's a little bit out of the scope of this actual bug fix

Reply:
np all good 👍